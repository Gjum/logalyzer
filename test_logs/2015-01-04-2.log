[10:09:38] [Server thread/INFO]: Starting minecraft server version 1.8.3
[10:09:38] [Server thread/INFO]: Loading properties
[10:09:38] [Server thread/INFO]: Default game type: SURVIVAL
[10:09:38] [Server thread/INFO]: Generating keypair
[10:09:39] [Server thread/INFO]: Starting Minecraft server on *:25565
[10:09:39] [Server thread/INFO]: Using epoll channel type
[10:09:39] [Server thread/INFO]: Preparing level "world"
[10:09:39] [Server thread/INFO]: Preparing start region for level 0
[10:09:41] [Server thread/INFO]: Preparing spawn area: 0%
[10:09:44] [Server thread/INFO]: Done (5.188s)! For help, type "help" or "?"

[10:09:45] [Server thread/INFO]: [Server] start

[11:00:00] [Server thread/INFO]: [Server] join HHL

[11:00:00] [User Authenticator #1/INFO]: UUID of player HHL is 808e727f-895a-4ac2-b246-4b6da2ca9451
[11:00:00] [Server thread/INFO]: HHL[/89.0.209.151:59550] logged in with entity id 1544982 at (-6392.5013284891775, 33.875, -934.0381456504325)
[11:00:00] [Server thread/INFO]: §6HHL§r joined the game

[12:00:00] [Server thread/INFO]: [Server] join Ulexos

[12:00:00] [User Authenticator #8/INFO]: UUID of player Ulexos is a1d94087-e9c0-412f-b90f-cff2c224d44f
[12:00:00] [Server thread/INFO]: Ulexos[/95.91.242.240:54254] logged in with entity id 517154 at (-6382.5, 44.0, -957.5)
[12:00:00] [Server thread/INFO]: §2Ulexos§r joined the game

[13:00:00] [Server thread/INFO]: [Server] leave HHL

[13:00:00] [Server thread/INFO]: §6HHL§r left the game
[13:00:00] [Server thread/INFO]: HHL lost connection: TextComponent{text='Disconnected', siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null}}

[14:00:00] [Server thread/INFO]: [Server] stop, was still on: Ulexos

[14:00:00] [Server thread/INFO]: Stopping server
[14:00:00] [Server thread/INFO]: Saving players
[14:00:00] [Server thread/INFO]: HHL lost connection: TranslatableComponent{key='disconnect.genericReason', args=[Internal Exception: java.lang.StackOverflowError], siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null}}
[14:00:00] [Server thread/INFO]: §6HHL§r left the game
[14:00:00] [Server thread/INFO]: Saving worlds
[14:00:00] [Server thread/INFO]: Saving chunks for level 'world'/Overworld
[14:00:00] [Server thread/INFO]: Saving chunks for level 'world'/Nether
[14:00:00] [Server thread/INFO]: Saving chunks for level 'world'/The End
