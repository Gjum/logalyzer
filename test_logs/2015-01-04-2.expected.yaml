first_event: 1420362578
last_event: 1420376400
online: {}
started: true
stopped: true
times:
- [9f352de0-b533-425c-9435-89b918ae4602, 1420095600, 1420340400, Offlinegott]
- [a1d94087-e9c0-412f-b90f-cff2c224d44f, 1420171200, 1420340400, Ulexos]
- [808e727f-895a-4ac2-b246-4b6da2ca9451, 1420365600, 1420372800, HHL]
- [a1d94087-e9c0-412f-b90f-cff2c224d44f, 1420369200, 1420376400, Ulexos]
